### PR TITLE
dev-java/jakarta-oro: EAPI 7

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: dev-java/jakarta-oro/jakarta-oro-2.0.8-r5.ebuild [hunk 1]
```diff
@@ -11,22 +11,22 @@ MY_J=${PN%%-*}
...
-DEPEND=">=virtual/jdk-1.6"
-RDEPEND=">=virtual/jre-1.6"
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
 
 S="${WORKDIR}/${P}"
 
 JAVA_SRC_DIR="src"
 
 java_prepare() {
```
Comment (fordfrog) [inline dev-java/jakarta-oro/jakarta-oro-2.0.8-r5.ebuild:28]: should be `src_prepare` when using eapi7

Comment (vaukai) [reply dev-java/jakarta-oro/jakarta-oro-2.0.8-r5.ebuild]: Ahh, sorry ...

**Comment by fordfrog** [comment]
@vaukai cool, thanks!

**Comment by vaukai** [comment]
@fordfrog 
Now or later: https://bugs.gentoo.org/780882

**Comment by fordfrog** [comment]
> @fordfrog
> Now or later: https://bugs.gentoo.org/780882

@vaukai done already yesterday, just did not update it here.

**Pull request closed without merging**